### PR TITLE
LPD-34272 - LocalFile.CommerceReplacementProducts#CanViewDiscontinuedReplacementSkusInProductDetails

### PR DESCRIPTION
**Pull request by brbalazs**
https://liferay.atlassian.net/browse/LPD-34272

This scenario is already covered by a test that is currently failing

cc @smottal

### CODE REVIEW

**Comment by brbalazs** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/frontend/data/set/provider/ReplacementCPInstanceFDSDataProvider.java [hunk 1]
```diff
@@ -68,7 +67,7 @@ public List<ReplacementSku> getItems(
 		long cProductId = ParamUtil.getLong(httpServletRequest, "cProductId");
 
 		if (sort == null) {
-			sort = SortFactoryUtil.getSort(CPInstance.class, "sku", "ASC");
+			sort = new Sort("sku", false);
```
Comment (smottal) [inline modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/frontend/data/set/provider/ReplacementCPInstanceFDSDataProvider.java:70]: `new Sort("sku", Sort.STRING_TYPE, false)` to be more aligned with the other usages of this

**Review by smottal — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/frontend/data/set/provider/ReplacementCPInstanceFDSDataProvider.java (hunk 1 above)
Comment (brbalazs) [reply modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/frontend/data/set/provider/ReplacementCPInstanceFDSDataProvider.java:70]: @smottal thanks, totally forgot to check how it's used in other places. Fixed it.

**Comment by brbalazs** [comment]
ci:test:sf

**Comment by brbalazs** [comment]
ci:test:relevant

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**